offchain-modules/packages/x/src/xchain/eth/contract.ts

```diff
@@ -103,6 +117,22 @@ export class EthChain {
     });
   }
 
+  async getUnlockLogs(
+    fromBlock: ethers.providers.BlockTag,
+    toBlock: ethers.providers.BlockTag,
+  ): Promise<{ log; parsedLog }[]> {
+    const logs = await this.provider.getLogs({
+      fromBlock: fromBlock,
+      address: ForceBridgeCore.config.eth.contractAddress,
+      topics: [unlockTopic],
```

Comment:
Can add unlockTopic to getLockLogs's filter topics

Reply:
keep original code after discussed